### ISSUE TITLE
Layer Inspector Mobile/Web: Sound panel should be aligned from the right side of layer inspector.

### ISSUE THREAD

**Issue by mericda** [irllabs/roundaround#245]
The sound panel should be right aligned so that app works in portrait mode as expected. 

![IMG_3647](https://user-images.githubusercontent.com/19254822/147646960-a6b968ac-e52b-4eda-a64f-2bb9162ddb71.jpg)

**Comment by mericda** [comment]
@muganwas did some further testing with different device sizes. Noticed that on devices like iPhone 6/7/8, Mixer panel doesn't get large as It can be. When I reviewed the code, I noticed that we use: 
```
@media (max-width: 959.95px)
<style>
.jss56 {
    top: -183px;
    left: 0;
    height: 180px;

```

Can we calculate the height of this modal in a different way (ex: fractional) so that the Mixer panel can be as high as the available area and potentially show more items/rounds? 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/19254822/148329654-b7dec8b7-404a-48a6-98a3-258700d024cf.png">
